dh-keys.js

```diff
@@ -3,20 +3,48 @@ const na = require('sodium-native')
 // this is for converting FeedKeys (ed25119 signing keys) into
 // Diffie-Hellman keys (curve25519 based shared key encryption)
 
+// na.randombytes_buf(buf)
```

Comment:
This doesn't look right? ;)

Reply:
Oh yeah, that was a snippet I was gonna use. Can tidy